### ISSUE TITLE
Question about training batchsize

### ISSUE THREAD

**Issue by LulinZhang** [pallashadow/SuperGlue-pytorch#8]
Hi, thank you very much for providing this useful repository.
If I understood correctly, you enabled training batchsize > 1 by padding the keypoints and descriptors with 0 value to reach the number of max_keypoints, right?
Wouldn't it influence the performance of the final model (because different patch pair has different number of padded points)? Should I do the padding also in the testing phase? 
Thank you in advance.

**Comment by pallashadow** [comment]
I think you are right. Padding is also needed in the testing phase.
